packages/fast-components-react-base/src/radio/radio.props.ts

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { ManagedClasses, RadioClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface RadioHandledProps {
```

Comment:
Can you change this to `export interface RadioHandledProps extends RadioManagedClasses`?